Makefile

```diff
@@ -11,22 +11,14 @@ BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
 ARTIFACTS := _out
 IMAGE_TAG ?= $(TAG)
 OPERATING_SYSTEM := $(shell uname -s | tr '[:upper:]' '[:lower:]')
-GOARCH := $(shell uname -m | tr '[:upper:]' '[:lower:]')
-
-ifeq ($(GOARCH),x86_64)
-  GOARCH := amd64
-endif
+GOARCH := $(shell uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/')
+SOURCE_DATE_EPOCH := $(shell git log -1 --pretty=%ct)
```

Comment:
i forgot to mention this change in kres, now we'll have proper source date epoch from the commit, so if someone checks build date, it's something reasonable